common/spacing/spacing-en.html

```diff
@@ -0,0 +1,120 @@
+</ul>
+
+<h3>Some working examples</h3>
+<p class="bg-primary mb-5">Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro, magnam.</p>
+<p class="bg-primary px-3">Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro, magnam.</p>
+<p class="bg-primary mt-sm-4">Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro, magnam.</p>
+<div class="d-flex">
+	<div class="col-xs-6 mx-auto"><p class="bg-primary">Lorem ipsum dolor sit amet.</p></div>
+</div>
+
+<h4>Code sample</h4>
+<pre><code>&lt;p class="bg-primary mb-5">Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro, magnam.&lt;/p>
+&lt;p class="bg-primary px-3">Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro, magnam.&lt;/p>
+&lt;p class="bg-primary mt-sm-4">Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro, magnam.&lt;/p>
+&lt;div class="d-flex">
+	&lt;div class="col-xs-6 mx-auto">&lt;p class="bg-primary">Lorem ipsum dolor sit amet.&lt;/p>&lt;/div>
+&lt;/div></code></pre>
+
+<h2>Additional spacing classes</h2>
+<h3><code>mrgn-{position}-{size}</code></h3>
+<p>Where <em>position</em> is one of:</p>
+<ul>
+	<li><code>tp</code> - for classes that set <code>margin-top</code></li>
+	<li><code>lft</code> - for classes that set <code>margin-left</code></li>
+	<li><code>rght</code> - for classes that set <code>margin-right</code></li>
+	<li><code>bttm</code> - for classes that set <code>margin-bottom</code></li>
+</ul>
+<p>Where <em>size</em> is one of:</p>
+<ul>
+	<li><code>0</code> - for classes that set a <code>margin</code> of <code>0px</code></li>
+	<li><code>sm</code> - for classes that set a <code>margin</code> of <code>5px</code></li>
+	<li><code>md</code> - for classes that set a <code>margin</code> of <code>15px</code></li>
+	<li><code>lg</code> - for classes that set a <code>margin</code> of <code>30px</code></li>
+	<li><code>xl</code> - for classes that set a <code>margin</code> of <code>50px</code></li>
+</ul>
+<h4>Some working examples</h4>
+<p class="bg-primary mrgn-tp-lg">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius, hic?</p>
+<p class="bg-primary mrgn-lft-md">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius, hic?</p>
+<p class="bg-primary mrgn-bttm-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius, hic?</p>
+<p class="bg-primary mrgn-rght-xl mrgn-bttm-sm">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius, hic?</p>
+<h5>Code sample</h5>
+<pre><code>&lt;p class="bg-primary mrgn-tp-lg">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius, hic?&lt;/p>
+&lt;p class="bg-primary mrgn-lft-md">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius, hic?&lt;/p>
+&lt;p class="bg-primary mrgn-bttm-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius, hic?&lt;/p>
+&lt;p class="bg-primary mrgn-rght-xl mrgn-bttm-sm">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius, hic?&lt;/p></code></pre>
+
```

Comment:
Est-ce que tu penses pouvoir ajouter un lien vers cette page non publié?

https://github.com/wet-boew/GCWeb/pull/2275/files

ou du moins une notes genre

```suggestion

<h2>WET-BOEW margin utility</h2>
<p>Working example to be added soon</p>
```

Reply:
@duboisp Je ne suis pas certain de comprendre. Tu veux que j'ajoute un lien vers Github ?